workflow-templates/im-test-playwright-manual.yml

```
@@ -0,0 +1,224 @@
+
+    steps:
+      - name: Calculate Ref to Checkout
+        id: calculate-ref
+        run: |
+          GITHUB_REF="${{ github.event_name == 'pull_request' && github.head_ref || env.DISPATCH_REF }}"
+          echo "GITHUB_REF=$GITHUB_REF" >> $GITHUB_ENV
+          echo "GITHUB_REF=$GITHUB_REF" >> $GITHUB_OUTPUT
+
+      - name: Annotate Inputs
+        run: |
+          echo $'
+          | Inputs                  | Value                                                                                     |
+          | ----------------------- | ----------------------------------------------------------------------------------------- |
+          | Environment             | `${{ inputs.environment }}`                                                               |
+          | Branch/Tag              | `${{ env.GITHUB_REF }}`                                                                   |
+          | Workflow Branch/Tag     | `${{ github.ref_name }}` - SHA: `${{ github.sha }}`                                       |
+          | Browser                 | `${{ inputs.browser-project }}`                                                           |
+          | Playwright Config File  | `${{ inputs.config-file }}`                                                               |
+          | Test Filter             | `${{ inputs.test-filter != 0 && inputs.test-filter || 'NA' }}`                            |
+          | Send Teams Notification | `${{ inputs.send-teams-notification == 0 && 'false' || inputs.send-teams-notification }}` |' >> $GITHUB_STEP_SUMMARY
+
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.GITHUB_REF }}
+
+      - name: Set Cache Keys
+        # As long as these files don't change the cache will remain in tact and the time
+        # to restore the different packages in the various jobs will be reduced.  If one
+        # of the files does change though, a new cache will be uploaded the next time
+        # it runs, then subsequent runs should be faster.
+        run: echo "NPM_CACHE_KEY=playwright_node_modules-${{ hashFiles('**/tests/playwright/package-lock.json') }}" >> $GITHUB_ENV
+
+      - name: Check for an npm cache
+        id: has-cache
+        uses: actions/cache@v4
+        with:
+          lookup-only: true
+          key: ${{ env.NPM_CACHE_KEY }}
+          path: |
+            ${{ env.NPM_CACHE_PATH }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        if: steps.has-cache.outputs.cache-hit != 'true'
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Authenticate with GitHub Packages
+        if: steps.has-cache.outputs.cache-hit != 'true'
+        uses: im-open/authenticate-with-gh-package-registries@v1
+        with:
+          read-pkg-token: ${{ secrets.READ_PKG_TOKEN }} # This is an org-level secret
+          orgs: 'im-client,im-enrollment,im-customer-engagement,im-practices,im-open'
```

I'd probably go even as far as removing two or three of these orgs. I don't know how often teams will need all five of them.
```suggestion
          orgs: 'im-client,im-enrollment,im-customer-engagement,im-practices,im-open'  # TODO: Verify list of orgs packages will be pulled from
```

---

I will just add the todo.  im-practices is used for the im.playwright-appinsights-reporter in an upcoming adr.